paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -154,7 +154,8 @@ void CpuPassStrategy::EnableMKLDNN() {
               "conv_bias_mkldnn_fuse_pass",    //
               "conv3d_bias_mkldnn_fuse_pass",  //
               "conv_elementwise_add_mkldnn_fuse_pass",
-              "conv_relu_mkldnn_fuse_pass"})) {
+              "conv_relu_mkldnn_fuse_pass",  //
+              "fc_mkldnn_pass"})) {
```

Comment:
Is fc_mkldnn_pass default ON?

Reply:
Yes, I have enabled it by default so that it can be functionally tested by `test_analyzers`.

Reply:
Is fc_mkldnn_pass default ON stabled for inference library?
If you want to test it by `test_analyzers`, how about add `cfg->pass_builder()->AppendPass(fc_mkldnn_pass)` for related unit-tests?

Reply:
Sure thing, I've done it in the most recent commit.